content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -17,6 +17,8 @@ When you turn maintenance mode on, RS:
 1. Takes a snapshot of the node configuration as a record of which shards and endpoints are on node at that time.
 1. Marks the node as a quorum node to prevent shards and endpoints from migrating into the node.
 1. Migrates shards to other nodes and binds endpoints to other nodes, if space is available on other nodes.
+1. Refuses requests for shard migration to the maintenance node.
```

Comment:
this is very similar to the 3rd point ("Marks the node as a quorum node...").
perhaps add this comment to that point?

Reply:
Good catch.